### PR TITLE
Replaces Nuka Cola in vendors (Alt PR to other nuka removal)

### PR DESCRIPTION
**Pull request by ghost**
:cl: SuicidalPickles
add: Added Strong-eBow, yet another radioactive drink, into the contraband list of drink vendors.
balance: Shambler's Juice has been added to premium drinks instead of contraband.
del: Nuka Cola has been removed from drink vendors.
/:cl:

got around to another replacement pr
basically the drink is 30 units apple juice and 10 uranium, comes in single stock, must have vendors hacked for it to be bought.

yes, you can just make nuka cola from this, but it'd be a lot less quantity considering this only comes stocked by single cans.

doesn't have any benefit, irradiates you

also shambler's was just added to premium drinks instead

capture

also i took this from /vg/

### CODE REVIEW

**Comment by wesoda25** [comment]
This towns only big enough for one nuka cola PR. Back off bee.

**Comment by RogueSteampunker** [comment]
Seems like a good compromise to me, rather than just removing it outright or replacing it with something silly like fernet.

**Comment by wesoda25** [comment]
> Seems like a good compromise to me, rather than just removing it outright or replacing it with something silly like fernet.

I disagree. All this does is just subtract 40 units of nuka cola.

**Comment by anconfuzedrock** [comment]
time to use uranium soda to emp brig
good compromise, keeps the new players who want to learn in the dark while nuka cola powergamers continue to powergame with their new drink that's slightly harder to learn to make

**Comment by ATH1909** [comment]
Nah, the real meta play is to use this uranium for virology so that you don't have to wait for the miners to bring back their first shipment of ore to start making your top-tier disease(s).

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -437,6 +437,17 @@
 	M.adjust_bodytemperature(-5 * TEMPERATURE_DAMAGE_COEFFICIENT, BODYTEMP_NORMAL)
 	..()
 
+/datum/reagent/consumable/strongebow
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: So, if the soda can of Strong-eBow just contains apple juice and uranium, why are you adding an unused reagent with a similar name?

**Comment by ExcessiveUseOfCobblestone** [comment]
you can make a max of 60u with a single glass of ebow (10u uranium). This is 10u more than just having the vending machine start with it.

Not to mention Virologists can now roundstart get very good symptoms.

I'm very hesitant on this solution.

**Comment by Suicidalpickles** [comment]
im retarded and forgot how chem works like that tbh

will probably just change it to the unused chem and make that irradiate you instead idk, just me and my shitcode

**Pull request closed without merging**